src/main/java/net/sf/jabref/sql/exporter/DatabaseExporter.java

```diff
@@ -305,28 +290,25 @@ private int populateGroupsTable(GroupTreeNode cursor, int parentID, int currentI
                 + (regExp != null ? '\'' + regExp + '\'' : "NULL") + ", " + hierContext.ordinal() + ", '" + database_id
                 + "');");
 
-        if(out instanceof Connection) {
-
-            // recurse on child nodes (depth-first traversal)
-            try (Statement statement = ((Connection) out).createStatement();
-                 ResultSet rs = statement.executeQuery(
-                    "SELECT groups_id FROM groups WHERE label='" + cursor.getGroup().getName() + "' AND database_id='"
-                            + database_id + "' AND parent_id='" + parentID + "';")) {
-                // setting values to ID and myID to be used in case of textual SQL
-                // export
-                int myID = currentID;
-                rs.next();
-                myID = rs.getInt("groups_id");
-                for (GroupTreeNode child : cursor.getChildren()) {
-                    ++currentID;
-                    currentID = populateGroupsTable(child, myID, currentID, out, database_id);
-                }
-                //Unfortunatley, AutoCloseable throws only Exception
-            } catch (Exception e) {
-                LOGGER.warn("Cannot close resource", e);
+        // recurse on child nodes (depth-first traversal)
+        try (Statement statement = ((Connection) out).createStatement();
+                ResultSet rs = statement.executeQuery(
+                        "SELECT groups_id FROM groups WHERE label='" + cursor.getGroup().getName() + "' AND database_id='"
+                                + database_id + "' AND parent_id='" + parentID + "';")) {
+            // setting values to ID and myID to be used in case of textual SQL
+            // export
+            int myID = currentID;
+            rs.next();
+            myID = rs.getInt("groups_id");
+            for (GroupTreeNode child : cursor.getChildren()) {
+                ++currentID;
+                currentID = populateGroupsTable(child, myID, currentID, out, database_id);
             }
-
+            //Unfortunatley, AutoCloseable throws only Exception
```

Comment:
Maybe take a look at what I did in #1265
I replaced all throws Exception with their typed ones